sentinelone_connector.py

```diff
@@ -1308,22 +1308,22 @@ def _handle_on_poll(self, param):
             container_id = self._create_container(threat)
             if not container_id:
                 continue
-            artifacts_creation_status, artifacts_creation_msg = self._create_artifacts(threat=threat, container_id=container_id)
+            artifacts_creation_status, artifacts_creation_message = self._create_artifacts(threat=threat, container_id=container_id)
             if phantom.is_fail(artifacts_creation_status):
-                self.debug_print(('Error while creating artifacts for container with ID {container_id}. {error_msg}').format(
-                    container_id=container_id, error_msg=artifacts_creation_msg))
+                self.debug_print(('Error while creating artifacts for container with ID {container_id}. {error_message}').format(
+                    container_id=container_id, error_message=artifacts_creation_message))
         self._state['first_run'] = False
         self._state['last_ingestion_time'] = end_time
         return action_result.set_status(phantom.APP_SUCCESS)
 
-    def _get_alerts(self, action_result, start_time, end_time, max_limit=None):
+    def _get_alerts(self, action_result, start_time, end_time, max_limit=1000):
```

Comment:
should we set this default to something smaller? like 50 or 100?

Reply:
I've considered that but felt that maintaining backward compatibility s better but I'm open to suggestion.

Reply:
on 2nd thought, I do like the idea of using 100 as default. 
I could add that as a default value in the APP JSON definition as well. 
Would that be acceptable?

Reply:
Should we put this fixed number in sentinelone_consts.py ?

Reply:
Agreed. 
I changed the limit to 100 and added the fix number in the const file. 
Also, updated APP JSON to flag unused on-poll action parameters.